Src/BootCamp.Chapter/MacFactory.cs

```diff
@@ -1,15 +1,75 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)